pyccl/tracers.py

```diff
@@ -744,7 +752,39 @@ def CMBLensingTracer(cosmo, z_source, n_samples=100):
     return tracer
 
 
-def tSZTracer(cosmo, z_max=6., n_chi=1024):
+def ZPowerTracer(cosmo, *, A, alpha, z_min=0., z_max=6., n_chi=1024):
```

Comment:
hmmm so Tracers that use this won't be instances of `ZPowerTracer`, but Tracers that use `NzTracer` will be instances of `NzTracer`.

I think the obvious solution, since this is a helper function, used to avoid repeated code, is to write it in the docs ("Helper that creates ..."), and rename it to something like `create_zpower_tracer` so that users won't confuse it for a class.

Or, (even better) add it as a `classmethod` to `Tracer`, so it can be called like `Tracer.from_zPower(...)`.